### PR TITLE
Add Code Coverage tool.

### PR DESCRIPTION
**Pull request by arrowplum**
https://aerospike.atlassian.net/browse/TOOLS-1599

### CODE REVIEW

**Comment by arrowplum** [comment]
@ClaytonKnittel if you `make coverage` it will run the tests and report test coverage locally. The numbers should be almost the same as the `codecov` tool but might be a little higher because the tool strips out some things like lone braces.

File: Makefile [hunk 1]
```diff
@@ -175,7 +125,6 @@ info:
 	@echo
 
 
-.PHONY: build
```
Comment (ClaytonKnittel) [inline Makefile:178]: How come the .PHONY's are removed? I thought they were to tell make to not expect a file by this name to be created as a product of this target

**Review by ClaytonKnittel — CHANGES_REQUESTED** [review]
Should we put the .PHONY's back (see comment on Makefile diff)?

File: Makefile (hunk 1 above)
Comment (arrowplum) [reply Makefile:178]: Yeah technically there is just one needed but I removed them in the process of debugging the make file. I should add them back in.

**Pull request merged**